org/code.go

```diff
@@ -1,26 +1,28 @@
-package tax
+package org
 
 import (
 	"regexp"
 
 	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/invopop/jsonschema"
 )
 
 // Code represents a string used to uniquely identify the data we're looking
 // at. We use "code" instead of "id", to reenforce the fact that codes should
 // be more easily set and used by humans within definitions than IDs or UUIDs.
-// Tax codes are standardised so that when validated they must contain between
+// Codes are standardised so that when validated they must contain between
 // 2 and 6 inclusive upper-case letters or numbers.
 type Code string
 
 var (
-	codeValidationRegexp = regexp.MustCompile(`^[A-Z0-9]{2,6}$`)
+	codePattern          = `^[A-Z0-9]{1,6}$`
+	codeValidationRegexp = regexp.MustCompile(codePattern)
 )
 
 // Validate ensures that the code complies with the expected rules.
 func (c Code) Validate() error {
 	return validation.Validate(string(c),
-		validation.Length(2, 6),
+		validation.Length(1, 6),
```

Comment:
I think we can omit this rule, because the regex pattern already includes the `{1,6}` quantifier. Alternatively, for consistency's sake, we could add a `{2,64} quantifier to the regexp pattern of `org.Key`.

Reply:
The main objetive was having the extra verbosity of the output, but you're right, it's not good to duplicate. I'll alter the regexp I think.